ios/PolyPodApp/PolyPod/ActivityViewController.swift

```diff
@@ -0,0 +1,18 @@
+import SwiftUI
+
+struct ActivityViewController: UIViewControllerRepresentable {
+    
+    var activityItems: [Any]
+    var applicationActivities: [UIActivity]? = nil
+    @Environment(\.presentationMode) var presentationMode
+    
+    func makeUIViewController(context: UIViewControllerRepresentableContext<ActivityViewController>) -> UIActivityViewController {
+        let controller = UIActivityViewController(activityItems: activityItems, applicationActivities: applicationActivities)
```

Comment:
Nice pragmatic solution to just use _share_ for now :D

Reply:
Well, at some point we should really look into updating the minVersion to iOS 14. SwiftUI got lots of improvements starting with iOS 14. We have lots of workarounds at the moment...